src/ngraph/graph_util.hpp

```diff
@@ -49,7 +49,7 @@ namespace ngraph
 
     void traverse_nodes(const Function* p,
                         std::function<void(std::shared_ptr<Node>)> f,
-                        bool include_control_deps);
+                        bool include_control_deps = false);
```

Comment:
If we're going to put a default here, I think `true` is more appropriate.

Reply:
Reverted change